snakemake/report/html_reporter/data/packages.py

```
@@ -38,14 +38,154 @@ def get_packages():
+            # End dependencies for vega-lite, included in vega-lite/vega-lite.js
+            # Any copied/derived/adapted code in vega-lite is shared with vega.
             "vega-embed": Package(
                 source_path="vega-embed/vega-embed.js",
                 license_path="vega-embed/LICENSE",
             ),
+            # Begin dependencies for vega-embed, included in vega-embed/vega-embed.js
+            # (excluding those shared with vega/vega-lite and therefore already documented)
+            "fast-json-patch": Package(
+                license_path="fast-json-patch/LICENSE",
+            ),
+            "json-stringify-pretty-compact": Package(
```

Ruff (0.8.2)

Dictionary key literal `"json-stringify-pretty-compact"` repeated

Remove repeated key literal `"json-stringify-pretty-compact"`

(F601)